js/OmniNav.js

```diff
@@ -25,20 +107,207 @@ var OmniNav = (function() {
 
   var buildNav = function(target) {
     console.log("Building OmniNav for target: " + target);
-    var $omniNav = $('<nav id="omni-nav" class="post-build" />');
-    var $logo = buildLogoLink();
-    $omniNav.append($logo);
+    var $omniNav = $('#omni-nav-v2');
+    var $primary = buildPrimaryNav();
+    $omniNav.append($primary);
     return $omniNav;
   };
 
-  var buildLogoLink = function() {
-    var $a = $('<a />').attr('id', 'omni-nav-logo')
-                       .attr('href', HOME_PAGE_URL)
-                       .attr('title', 'Chapman University Website Home Page');
-    var $img = $('<img />').attr('src', LOGO_URL);
-    $a.append($img);
-    return $a;
-  };
+  /* PRIMARY NAV */
+  var buildPrimaryNav = function() {
+    var $primary = $('<div />').attr('id', 'primary-nav');
+
+    // Left Container
+    var $leftContainer = buildLeftNavContainer();
+
+    // Logo Container
+    var $logo = buildLogoContainer();
+
+    //Right Container
+    var $rightContainer = buildRightNavContainer();
+
+    //Global Nav Container
+    var $globalNav = buildGlobalNav();
+
+    $primary.append($leftContainer, $logo, $rightContainer, $globalNav);
+    return $primary;
+  }
+
+  var buildLeftNavContainer = function() {
+    var $leftContainer = $('<div />').attr('class', 'nav-container left-nav-container');
+    var $offCanvasTrigger = $('<a />').attr({
+        class: 'off-canvas-trigger primary-nav-icon',
+        href: '#',
+        id: 'js-off-canvas-trigger',
+        role: 'button',
+        title: 'Access links to the pages within this section of the website and to other sections of the website',
+        'aria-label': 'Access links to the pages within this section of the website and to other sections of the website'
+    });
+
+    var $offCanvasIcon = buildOffCanvasSVG();
+    $offCanvasTrigger.append($offCanvasIcon);
+    $leftContainer.append($offCanvasTrigger);
+    return $leftContainer;
+  }
+
+  var buildLogoContainer = function() {
+    var $logoContainer = $('<div />').attr('class', 'nav-container logo-container');
+    var wrapper = $('<div />').attr('class', 'cu-logo-wrapper');
+    var primaryLogo = $('<div />').attr('class', 'primary-logo');
+    var logoLink = $('<a />').attr({
+      href: HOME_PAGE_URL,
+      title: "Chapman University Website Home Page"
+    });
+    var svgLogo = buildChapmanLogoSVG();
+    logoLink.append(svgLogo);
+    primaryLogo.append(logoLink);
+    wrapper.append(primaryLogo);
+    return $logoContainer.append(wrapper);
+  }
+
+  // Contains search icon (utility nav trigger) and login icon (login menu dropdown)
+  var buildRightNavContainer = function() {
+    var $rightContainer = $('<div />').attr('class', 'nav-container right-nav-container');
+
+    // Login Menu Dropdown
+    var loginTrigger = $('<div />').attr('class', 'primary-nav-action login-trigger');
```

Comment:
Should these variables begin with $? I noticed that naming convention for the other jquery object variables.